e2e-tests/insurance/cypress/e2e/journeys/policy/broker-addresses/broker-addresses-page.spec.js

```diff
@@ -38,7 +47,34 @@ context('Insurance - Policy - Broker addresses page', () => {
       pageTitle: CONTENT_STRINGS.PAGE_TITLE,
       currentHref: `${ROOT}/${referenceNumber}${BROKER_ADDRESSES_ROOT}`,
       backLink: `${ROOT}/${referenceNumber}${BROKER_ADDRESSES_ROOT}#`,
-      assertSaveAndBackButtonDoesNotExist: true,
+    });
+  });
+
+  describe('page tests', () => {
+    beforeEach(() => {
+      cy.navigateToUrl(url);
+    });
+
+    it('renders body copy', () => {
+      const expected = `1 ${CONTENT_STRINGS.BODY} W1A 1AA`;
+
+      cy.checkText(body(), expected);
+    });
+
+    describe(FIELD_ID, () => {
+      const field = fieldSelector(FIELD_ID);
+
+      it('renders a legend', () => {
+        cy.checkText(field.label(), FIELD_STRINGS[FIELD_ID].LABEL);
+      });
+
+      it('renders an `address` radio label and input', () => {
```

Comment:
Note: For this PR, there is only 1x test for the address results because we are using a hard coded postcode to fetch the data.

In a future PR(s), this will be dynamic and could render for example 20 addresses.

Reply:
There is a ticket specifically for this EMS-3991

Reply:
Idea at the moment is to make an Ordnance survey API call in E2E and then map over the results and run cypress assertions